src/components/Proposal/ProposalContainer.tsx

```diff
@@ -401,7 +409,9 @@ class ProposalContainer extends React.Component<IProps, IState> {
 
                 ON {submittedTime.format("MMM DD, YYYY")}
               </div>
-
+              <Link to={"/dao/" + dao.avatarAddress + "/proposal/" + proposal.proposalId}>
```

Comment:
Where does this show up? I don't see it in the review app ([this proposal](https://daostack-alchemy-stagin-pr-187.herokuapp.com/#/dao/0x141dfb78b3c5a81d8273720d830179afeb8bb0b9/proposal/0x7fe799021724a430356a0bbb63de43d966d56b141e411b98adba0cdae2c8bedb))

Reply:
huh, thats weird, it works locally

Reply:
huh, actually it only appears after you have visited the proposal page for the first time. maybe thats ok

Reply:
yeah, someone has to visit the page to create the disqus conversation